File: avalon-support/avalon-oauth2-support/avalon-oauth2-authorization-server/src/test/java/io/github/lishangbu/avalon/oauth2/authorizationserver/keygen/UuidKeyGeneratorTest.java
package io.github.lishangbu.avalon.oauth2.authorizationserver.keygen;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotNull;
import static org.junit.jupiter.api.Assertions.assertTrue;

import org.junit.jupiter.api.Test;

class UuidKeyGeneratorTest {

    @Test
    void generatesLowercaseUuid() {
        UuidKeyGenerator generator = new UuidKeyGenerator();

        String key = generator.generateKey();

        assertNotNull(key);
        assertEquals(key.toLowerCase(), key);
        assertTrue(key.contains("-"));
    }
}

File: avalon-platform/avalon-security/avalon-oauth2-authorization-server/src/test/kotlin/io/github/lishangbu/avalon/oauth2/authorizationserver/keygen/UuidKeyGeneratorTest.kt
package io.github.lishangbu.avalon.oauth2.authorizationserver.keygen

import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test

class UuidKeyGeneratorTest {
    @Test
    fun generatesLowercaseUuid() {
        val generator = UuidKeyGenerator()

        val key = generator.generateKey()

        assertNotNull(key)
        assertEquals(key.lowercase(), key)
        assertTrue(key.contains("-"))
    }
}
